client/my-sites/plugins/plugin-list-header/index.jsx

```diff
@@ -171,8 +171,8 @@ export default React.createClass( {
 						<Button compact primary onClick={ this.props.updateAllPlugins } >
 							{
 								this.translate(
-									'Update %(numUpdates)d Plugin',
-									'Update %(numUpdates)d Plugins',
+									'Updated %(numUpdates)d Plugin',
```

Comment:
Hi! I've found a possible matching string that has already been translated **67** times:
`translate( '%d Plugin Update', '%d Plugin Updates' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).